HousingSearchApi.Tests/V1/TestHelpers/CreateRandomISOStartOfTenureDateTests.cs

```diff
@@ -0,0 +1,53 @@
+using AutoFixture;
+using System;
+using System.Globalization;
+using Xunit;
+using QueryableTenure = Hackney.Shared.HousingSearch.Gateways.Models.Tenures.QueryableTenure;
+
+namespace HousingSearchApi.Tests.V1.TestHelpers
+{
+    public class CreateRandomISOStartOfTenureDateTests
+    {
+        private readonly Fixture _fixture = new();
+
+        //This is the date time format currently used in the index
+        private readonly string _dateFormat = "yyyy-MM-ddTHH:mm:ssZ";
+        private readonly CultureInfo _provider = CultureInfo.InvariantCulture;
+
+        public CreateRandomISOStartOfTenureDateTests()
+        {
+            _fixture.Customize(new CreateRandomISOStartOfTenureDate());
+        }
+
+        [Fact]
+        public void CreateRandomISOStartOfTenureDateReturnsDateInTeCorrectFormat()
```

Comment:
Spelling mistake with word `the`.

Reply:
This test was deleted as part of the changes made in [642e7e2](https://github.com/LBHackney-IT/housing-search-api/pull/224/commits/642e7e23a648dbb325df8c2dc925aca7c33fffae)